### PR TITLE
Migrate to README.yaml format

### PR DESCRIPTION
**Pull request by vadim-hleif**
## what
* Add `README.yaml`

## why
* Standardize README

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -1,31 +1,39 @@
 output "cf_id" {
-  value = "${aws_cloudfront_distribution.default.id}"
+  value       = "${aws_cloudfront_distribution.default.id}"
+  description = "ID of AWS CloudFront distribution"
 }
 
 output "cf_arn" {
-  value = "${aws_cloudfront_distribution.default.arn}"
+  value       = "${aws_cloudfront_distribution.default.arn}"
+  description = "ID of AWS CloudFront distribution"
```
Comment (sarkis) [inline outputs.tf]: ARN of AWS CloudFront distribution

File: variables.tf [hunk 2]
```diff
@@ -32,7 +32,8 @@ variable "tags" {
 }
 
 variable "enabled" {
-  default = "true"
+  default     = "true"
+  description = "State of CloudFront"
```
Comment (sarkis) [inline variables.tf]: Boolean which determines if CloudFront resources are created

File: variables.tf [hunk 3]
```diff
@@ -110,39 +120,47 @@ variable "log_expiration_days" {
...
+  default     = "false"
+  description = "Forward query strings to the origin that is associated with this cache behavior"
 }
 
 variable "cors_allowed_headers" {
-  type    = "list"
-  default = ["*"]
+  type        = "list"
+  default     = ["*"]
+  description = "List of allowed headers  for S3 bucket"
```
Comment (sarkis) [inline variables.tf]: unneeded extra space between words `headers` and `for`

File: variables.tf [hunk 4]
```diff
@@ -110,39 +120,47 @@ variable "log_expiration_days" {
...
 }
 
 variable "cors_allowed_methods" {
-  type    = "list"
-  default = ["GET"]
+  type        = "list"
+  default     = ["GET"]
+  description = "List of allowed methods (e.g. ` GET, PUT, POST, DELETE, HEAD`) for S3 bucket"
```
Comment (sarkis) [inline variables.tf]: Do we need the backtick here? If so maybe we should do e.g. `GET`, `PUT`, ...

File: variables.tf [hunk 5]
```diff
@@ -110,39 +120,47 @@ variable "log_expiration_days" {
...
 }
 
 variable "cors_allowed_origins" {
-  type    = "list"
-  default = []
+  type        = "list"
+  default     = []
+  description = "List of allowed origins (e.g. ` example.com, test.com`) for S3 bucket"
```
Comment (sarkis) [inline variables.tf]: do we need the ` here?

File: variables.tf [hunk 6]
```diff
@@ -151,45 +169,54 @@ variable "viewer_protocol_policy" {
 }
 
 variable "allowed_methods" {
-  type    = "list"
-  default = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+  type        = "list"
+  default     = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+  description = "List of allowed methods (e.g. ` GET, PUT, POST, DELETE, HEAD`) for AWS CloudFront"
```
Comment (sarkis) [inline variables.tf]: do we need the ` here?

File: variables.tf [hunk 7]
```diff
@@ -151,45 +169,54 @@ variable "viewer_protocol_policy" {
...
 }
 
 variable "cached_methods" {
-  type    = "list"
-  default = ["GET", "HEAD"]
+  type        = "list"
+  default     = ["GET", "HEAD"]
+  description = "List of cached methods (e.g. ` GET, PUT, POST, DELETE, HEAD`)"
```
Comment (sarkis) [inline variables.tf]: do we need the ` here?

File: variables.tf (hunk 6 above)
Comment (osterman) [reply variables.tf]: yea, let's just remove the back tick

**Pull request merged**